scripts/west_commands/runners/linkserver.py

```diff
@@ -169,7 +169,7 @@ def do_erase(self, **kwargs):
         else:
             _cmd_core = ""
 
-        linkserver_cmd = ([self.linkserver, "flash"] + ["--probe", "#"+str(self.probe)] +
+        linkserver_cmd = ([self.linkserver, "flash"] + ["--probe", str(self.probe)] +
```

Comment:
This is fine. However, 

 - if we keep the same switch (i.e. --probe), then the help text for the linkserver runner should be updated, as it clearly excludes the use of the probe's serial number

Reply:
Let me update the document as well